### PR TITLE
feat(APM): elastic APM added

### PR DESCRIPTION
**Pull request by EdwinRozario**
Airbreak and all of its references removed

### CODE REVIEW

File: config/elastic_apm.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+server_url: <%= ENV.fetch('ELASTIC_APM_SERVER_URL', 'localhost:8200') %>
+service_name: "Lending Services"
```
Comment (Reinkaos) [inline config/elastic_apm.yml:2]: Lending Services 😂

**Review by Reinkaos — CHANGES_REQUESTED** [review]
Change the app name bro :D

File: config/elastic_apm.yml (hunk 1 above)
Comment (EdwinRozario) [reply config/elastic_apm.yml]: Done!

**Review by Reinkaos — APPROVED** [review]
🥇

**Pull request merged**